node/resolvers/benefits/index.ts

```diff
@@ -1,110 +1,110 @@
 import axios from 'axios'
-import paths from '../paths'
 import { flatten } from 'ramda'
-import { withAuthToken } from '../headers'
+
 import { resolveLocalProductFields } from '../catalog/fieldsResolver'
+import { queries as checkoutQueries } from '../checkout'
+import { withAuthToken } from '../headers'
+import paths from '../paths'
 
-/**
- * Default values for Shipping Items
- */
-const DEFAULT_QUANTITY = '1'
+const SKU_SEPARATOR = ','
+const CATALOG = 'Catalog'
 const DEFAULT_SELLER = '1'
+const DEFAULT_QUANTITY = '1'
 
 /**
- * Retrieve a list of SKU Items.
- * @param skuIds List of the skuIds of the skus that must be retrieved.
- * @param ioContext VTEX ioContext which contains the informations of the application.
+ * Receives an Array of SKU Ids and returns an Array of the Products that each SKU Belongs.
+ *
+ * @param skuIds Array of SKU Ids which will be used to retrieve all the products that each SKU belongs.
+ * @param ioContext Helper object which holds the account and the authentication headers.
  */
-const getSKUItems = async (skuIds, ioContext) => {
-  return await Promise.all(skuIds.map(async skuId => {
-    const { data } = await axios.get(paths.skuById(ioContext.account, { skuId }), { headers: { Authorization: ioContext.authToken }})
-    return data
-  }))
+const resolveProducts = async (skuIds, ioContext) => {
```

Comment:
It would be better to type, instead of commenting the params

Reply:
When you say type you mean a single text that explains all about the function including the params?